recipe/meta.yaml

```diff
@@ -37,19 +39,25 @@ test:
     - rope.refactor.importutils
   requires:
     - pip
-    - python <3.10
+    - python
+    - pytest
+    - platformdirs
```

Comment:
The `pip check` failure points to that `pytoolconfig[global]` is needed. This optional dependency requires `platformdirs` to be added as a dependency to `pytoolconfig`. So, `platformdirs` should at least be added as a `run` dependency, not just a test dependency.

However, you may need to rebuild `pytoolconfig` with `platformdirs` to get the `pip check` command to work.